### DISCUSSION TITLE
Can't disable replay cache

### DISCUSSION THREAD

**Discussion by jczuchnowski** [stnoonan/spnego-http-auth-nginx-module#117] [Q&A] [answered]
I need to disable the replay cache on Nginx server. My build is using `libkrb5-dev` so it's an MIT Kerberos. Following the documentation on https://web.mit.edu/kerberos/krb5-latest/doc/basic/rcache_def.html I'm setting env var `KRB5RCACHETYPE=none`. Unfortunately it doesn't change anything. The replay cache is still created as `/var/tmp/http_33` and subsequent requests are rejected with 403 (and `Request is a replay` in the logs).

I'm building a docker image with:
- Debian 10.1
- Nginx 1.17.4
- libkrb5-dev 1.17-3

Any ideas on what I'm doing wrong?

**Comment by stnoonan** [comment, marked as answer]
Sorry, no idea. Why would you want to disable the replay cache?

**Comment by Dsolnik** [comment]
You likely needed to add 
```
env KRB5RCACHETYPE;
```
to the top of your nginx.conf
